sankeerth/java8features/pom.xml

```diff
@@ -0,0 +1,25 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.yash.training</groupId>
+  <artifactId>java8features</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>java8features</name>
+  <url>http://maven.apache.org</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
```

Comment:
explain different scopes in maven and life cycle of a maven build process

Reply:
There are 6 different scopes in maven.

compile - This is the default scope. If none of the scopes are specified, this scope will be used. 

provided - This is much like compile, but indicates you expect the JDK or a container to provide the dependency at runtime. 

runtime - This scope indicates that the dependency is not required for compilation, but is for execution.

test - This scope indicates that the dependency is not required for normal use of the application, and is only available for the test compilation and execution phases.

system - This scope is similar to "provided" except that you have to provide the JAR which contains it explicitly. 

import - This scope is only used on a dependency of type pom in the section. It indicates that the specified POM should be replaced with the dependencies in that POM's section.